dotnet/CoreLib/Pipeline/MimeTypes.cs

```diff
@@ -23,48 +28,61 @@ public static class FileExtensions
     public const string MsWordX = ".docx";
     public const string Pdf = ".pdf";
     public const string TextEmbeddingVector = ".text_embedding";
+    public const string ImageJpeg = ".jpeg";
+    public const string ImageJpg = ".jpg";
+    public const string ImagePng = ".png";
+    public const string ImageTiff = ".tiff";
```

Comment:
Do we want to expand the types here? (.bmp? .gif? ...)

Reply:
only if needed, a few won't hurt :-)

Reply:
I'm just anticipating a user asking why A but not B (for common formats)...agree don't want to go crazy here.  I like the content detection approach you've described for the future.